src/test/java/org/galatea/starter/entrypoint/IexRestControllerTest.java

```diff
@@ -81,4 +80,107 @@ public void testGetLastTradedPriceEmpty() throws Exception {
         .andExpect(jsonPath("$", is(Collections.emptyList())))
         .andReturn();
   }
+
+  @Test
+  public void testGetHistoricTradedPricesWithDate() throws Exception {
+    MvcResult result = this.mvc.perform(
+                    org.springframework.test.web.servlet.request.MockMvcRequestBuilders
+                            .get("/iex/historicalTradedPrices?symbols=NET&date=20230408")
+                            .accept(MediaType.APPLICATION_JSON_VALUE))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$[0][0].symbol", is("NET")))
+            .andReturn();
+  }
+
+  @Test
+  public void testGetHistoricTradedPricesWithDate2() throws Exception {
+    MvcResult result = this.mvc.perform(
+                    org.springframework.test.web.servlet.request.MockMvcRequestBuilders
+                            .get("/iex/historicalTradedPrices?symbols=NET,TSLA&date=20230408")
+                            .accept(MediaType.APPLICATION_JSON_VALUE))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$[0][0].symbol", is("NET")))
```

Comment:
Same here, we should validate every field is returned correctly. Best to avoid copy/pasting if possible and extract validations like these to their own method.